pkg/api/admin.go

```diff
@@ -35,8 +36,8 @@ func (hs *HTTPServer) getAuthorizedSettings(ctx context.Context, user *models.Si
 		return bag, nil
 	}
 
-	eval := func(scopes ...string) (bool, error) {
-		return hs.AccessControl.Evaluate(ctx, user, accesscontrol.ActionSettingsRead, scopes...)
+	eval := func(scope string) (bool, error) {
```

Comment:
I suppose building the evaluation chain to simplify the following code of getAuthorizedSettings, is part of another PR?

Reply:
no, we are filtering properties of settings based on which keys you have access to. If we would evaluate all in one go we would not know what scopes we are missing

Reply:
My question was more: how does an admin know what permission the end user was lacking? I'm having a hard time understanding the point of returning the error message ID if there is no additional value to it.

Reply:
I understand. We are not trying to see what the user has access to not whether the user has access to all or part of the resources. Thanks :slightly_smiling_face: